pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>org.apache.mesos</groupId>
     <artifactId>hadoop-mesos</artifactId>
-    <version>0.1.0</version>
+    <version>0.1.1-SNAPSHOT</version>
```

Comment:
This is the first step to getting into a release process with Maven. It's a bit counterintuitive at first, ```SNAPSHOT``` means "the versions that lead up to the release without the 'SNAPSHOT' classifier". So versions leading up to 0.1.1 will be 0.1.1-SNAPSHOT, users should expect them to be evolving in nature. The release plugin will take care of removing ```SNAPSHOT``` when it's time to release. I'll set that up in a follow-on PR.